setup/module/api/core.tf

```diff
@@ -178,6 +178,6 @@ data "aws_ami" "windows" {
 
   filter {
     name   = "name"
-    values = ["Windows_Server-2016-English-Full-ECS_Optimized-2017.11.24"]
+    values = ["Windows_Server-2016-English-Full-ECS_Optimized-2018.03.26"]
```

Comment:
I did find a more recent Windows AMI dated at 2018.04.18, unless we have a reason to use this March release.